README.md

```diff
@@ -1,2 +1,86 @@
-# Product-Landing-Page
-This is our fourth real project 
+
+
+## Product-Landing-Page 📝
+
+> ## Project Description 📝
+project landing page  this project is  abuot mekup product i create this that if someone want they can buy just by click shop now and enter email .
+> 
+
+
+<!-- Include code snippets or examples to demonstrate your project. -->
+
+/* .wrapper {
+  background-color: white;
+  width: 90%;
+  height: 90vh;
+} */
+
+//<nav class="nav-bar">
+        <a class="nav-link" id="Features" href="#Features">Features</a>
+        <li><a class="nav-link" href="#Features">Home</a></li>
+        <li><a class="nav-link" href="#Features">Orders</a></li>
+        <li><a class="nav-link" href="#Features">Product</a></li>
+        <li><a class="nav-link" href="#Features">Contact</a></li>
+      </nav> .
+
+## Demo 📸
+
+Include a demo or animated GIF of your project. You can use tools like [asciinema](https://asciinema.org/) to record your terminal sessions.
+
+![Demo](<img width="913" alt="Screenshot 2024-05-31 040232" src="https://github.com/royajamal/Product-Landing-Page/assets/168343975/c1b4db77-894c-4fe0-a67e-d719a09db2cb">)
```

Comment:
- You added an image tag, remove it and add a screenshot of your project instead.
- Also remove the instruction text.